app/scripts/controllers/tables.js

```diff
@@ -108,7 +108,7 @@ angular.module('tables', ['stats', 'sql', 'common', 'tableinfo', 'events'])
 
         const PARTITION_QUERY = 'SELECT partition_ident, number_of_shards, number_of_replicas, values ' +
           'FROM information_schema.table_partitions ' +
-          'WHERE schema_name = ? AND table_name = ? AND closed = false';
+          'WHERE table_schema = ? AND table_name = ? AND closed = false';
```

Comment:
is this a breaking change ? it's missing a Changes.txt entry

Reply:
Thanks, it is, I will add it.